### DISCUSSION TITLE
Some crude ensemble estimates for noice part

### DISCUSSION THREAD

**Discussion by anddebol** [ARM-Development/comble-mip#59] [Show and tell]
Hi everyone,

Our group was tinkering with some estimates of inherit ensemble spread and differences between LES models on warm (noice) microphysics. 
To estimate the ensemble spread we ran 8 members with different initial perturbation of inside boundary layer temperature, and also some experiments with the same initial conditions but different timesteps for lw radiation (nskip in the legend below, we run RRTMG for that in RCC MSU LES). Note that those experiments are on coarse grid (200m horizontally and 75 levels with uniform vertical resolution -- way more forgiving in terms of computational costs). 
Some of the results are in figure below
![clt_lwpc_rlut](https://github.com/ARM-Development/comble-mip/assets/4727244/b21abc08-a3a0-42ac-b5d3-5c25e8f93ed7)

Basically two things can be noted here: prescribed resolution run falls inside the ensemble spread, and most of the differences between the different LES models are larger than the spread (though they get  closer towards the end of the experiment for LWP).  However the amount of ensemble members definitely needs to be increased, for anything definitive to say.

**Comment by anddebol** [comment]
Quick followup 
for LWP-rain it's definitely the case that ensemble spread on coarse model overlaps inter-model variance a lot 
![lwr](https://github.com/ARM-Development/comble-mip/assets/4727244/c4bd81ba-edde-4d72-a121-4e0d5b6608cf)

**Comment by fridlind** [comment]
Thank you for sharing this! We focused most of our attention so far on the ice-containing (FixN) simulations, where the small-domain simulations develop oscillations of domain-mean quantities that seem to be associated with a dominating precipitating cell lifecycle. Would it make sense to consider a longer averaging time to span several oscillations that may be associated with typical cell lifecycles?

**Reply by anddebol** [reply]
Hi, sorry for the delayed reply.  From what we can gather from est runs ensemble spread for ice-containing experiments would be of the same magnitude (or bigger for species like graupel).  Limiting ensemble spread with longer time averaging comes might complicate analysis in growing in time CBL, so I think  larger  domain will be more efficient in that.
